src/Components/NewClientForm.js

```diff
@@ -0,0 +1,240 @@
+                name="kindness"
+                placeholder="Kindness"
+                min={ 0 } 
+	            max={ 10 } 
+                value={kindness}
+                onChange={event => setKindness(event.target.value)}
+            /> 
+        <label for="vibe">Vibe</label>
+        <input
+                id="vibe"
+                type="range"
+                name="vibe"
+                placeholder="Vibe"
+                min={ 0 } 
+	            max={ 10 } 
+                value={vibe}
+                onChange={event => setVibe(event.target.value)}
+            /> 
+        <label for="date-again">Date Again</label>
+         <select id="dateAgain" name="date-again" onChange={event => setDateAgain(event.target.value)}>
+           <option value="">Would You Date Them Again?</option>
+           <option value="Yes">Yes</option>
+           <option value="No">No</option>
+         </select>
+         {/* <label for="kinks">Kinks</label> */}
+         {/* <input
+                id="kinks"
+                type="text"
+                name="kinks"
+                placeholder="Kinks/Activities?"
+                value={kinks}
+                onChange={event => setKinks(event.target.value)}
+            />
+          <label for="hygiene">Hygiene</label>
+         <input
+                id="hygiene"
+                type="text"
+                name="hygiene"
+                placeholder="How Was Their Hygiene?"
+                value={hygiene}
+                onChange={event => setHygiene(event.target.value)}
+            />
+          <label for="condom-usage">How Willing Were They To Use Condoms?</label>
+          <input
+                id="condomUsage"
+                type="text"
+                name="condom-usage"
+                placeholder="Condom Use Willingness"
+                value={condomUsage}
+                onChange={event => setCondomUsage(event.target.value)}
+            />
+        <label for="date-duration">How Long Was the Date?</label>
+           <input
+                id="dateDuration"
+                type="range"
+                name="date-duration"
+                min={ 1 } 
+	            max={ 5 } 
+                placeholder="How Long was the Date?"
+                value={duration}
+                onChange={event => setDuration(event.target.value)}
+            />
+        <label for="punctuality">How Punctual Were They?</label>
+         <input
+                id="punctuality"
+                type="text"
+                name="punctuality"
+                placeholder="How Punctual Were They?"
+                value={punctuality}
+                onChange={event => setPunctuality(event.target.value)}
+            /> */}
+            <NavLink to={`/Search`}>
+                <button id="submitBtn" className="submit-button" onClick={(event) => submitReview(event)}>Submit New Review{(!rating || !body || !safetyMeter || !email) && <p>{missingInfo}</p>}</button>
```

Comment:
we can remove event here and in the submitReview func above since we are no longer using event.preventDefault()